static/bidder-info/interactiveOffers.yaml

```diff
@@ -0,0 +1,10 @@
+maintainer:
+  email: "faria@interactiveoffers.com"
+gvlVendorID: 0
```

Comment:
I recommend removing the `gvlVendorID` line entirely if you aren't registered with the IAB Vendor List.

Reply:
gvlVendorId was removed